net/macos-vpn-server/Portfile

```diff
@@ -0,0 +1,157 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+
+name                    macos-vpn-server
+# use the latest macOS version as the version number of this port
+version                 10.14.5
+homepage                https://opensource.apple.com/source/ppp/ppp-838/Helpers/vpnd/vpnd.5.auto.html
+categories              net
+platforms               darwin
+supported_archs         noarch
+maintainers             @essandess
```

Comment:
Can you add your email and `openmaintainer` here?

Reply:
Done, thanks.